pkg/blockstorage/zone/zone.go

```diff
@@ -28,6 +28,8 @@ import (
 	"github.com/kanisterio/kanister/pkg/kube"
 	kubevolume "github.com/kanisterio/kanister/pkg/kube/volume"
 	"github.com/kanisterio/kanister/pkg/log"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Could skip `v1` alias here

Reply:
3rd party imports should be in the second import group. Could you move these above please?